spec/concurrent/array_spec.rb

```diff
@@ -14,5 +14,15 @@ module Concurrent
         end
       end.map(&:join)
     end
+
+    describe '#slice' do
+      # This is mostly relevant on Rubinius and Truffle
```

Comment:
FWIW TruffleRuby tries to avoid extra public methods, so new_range/new_reserved are not defined on Array in TruffleRuby.
Arguably, this is a Rubinius bug to expose internal methods as public.